README.md

```diff
@@ -1,5 +1,11 @@
 # deep-dircmp
 
+![codeql](https://github.com/BercziSandor/python-deep-dircmp/actions/workflows/codeql.yml/badge.svg)
```

Comment:
This is invalid URL for this repository.

Reply:
Totally valid point, fixed.

Reply:
I do not see it yet fixed, please ping me once it is.